fhir-database-utils/src/main/java/com/ibm/fhir/database/utils/db2/Db2Adapter.java

```diff
@@ -105,15 +105,15 @@ public void createIntVariable(String schemaName, String variableName) {
     }
 
     @Override
-    public void createPermission(String schemaName, String permissionName, String tableName, String predicate) {
+    public void createOrReplacePermission(String schemaName, String permissionName, String tableName, String predicate) {
         final String qualifiedPermissionName = DataDefinitionUtil.getQualifiedName(schemaName, permissionName);
         final String qualifiedTableName = DataDefinitionUtil.getQualifiedName(schemaName, tableName);
         DataDefinitionUtil.assertSecure(predicate);
 
         final String ddl = ""
-                + "       CREATE PERMISSION " + qualifiedPermissionName
-                + "                      ON " + qualifiedTableName
-                + "          FOR ROWS WHERE " + predicate
+                + "CREATE OR REPLACE PERMISSION " + qualifiedPermissionName
```

Comment:
Interesting... it's left aligned in the output. 

```
CREATE PERMISSION FHIRAPP.STR_VALUES_TENANT                      ON FHIRAPP.STR_VALUES          FOR ROWS WHERE FHIRAPP.STR_VALUES.MT_ID = FHIR_ADMIN.SV_TENANT_ID ENFORCED FOR ALL ACCESS ENABLE ;
```

Reply:
i think this format is wonky...i just tried preservign what was there

Reply:
actually, now i can't seem to find where i inhereted this from...will change

Reply:
so sorry it was an observation - not intended to change.

Reply:
updated